java/meterpreter/stdapi/src/main/java/com/metasploit/meterpreter/stdapi/stdapi_ui_send_keys.java

```diff
@@ -0,0 +1,5 @@
+package com.metasploit.meterpreter.stdapi;
```

Comment:
@mb-syss I don't think this file was meant to be included. You can probably remove it relatively painlessly by rebasing (e,g `git pull -r origin master`)